AmericanExpress/src/test/java/com/braintreepayments/api/AmericanExpressAnalyticsUnitTest.java

```diff
@@ -0,0 +1,18 @@
+package com.braintreepayments.api;
```

Comment:
Curious what the plan is for adding new files in Java vs Kotlin? Are we waiting to add anymore Kotlin until Post v5?

Reply:
We don't have an official plan yet, but these can be in Kotlin since it's cleaner that way - converted them and the tests.